ieeg_filtered_speech/sub-cm4/ses-01/ieeg/sub-cm4_task-filtered-speech_run-01_space-2DImage_electrodes.tsv

```diff
@@ -0,0 +1,65 @@
+type	x	y	name
+grid	421.0	85.0	G1
```

Comment:
type must be one of surface/depth/dbs and add columns for size of the electrode contacts

Reply:
good call - need to track this down as it's not reported in my paper (and apparently none of the other knight lab papers I randomly sampled either :-P )